website/www/site/layouts/index.html

```diff
@@ -49,6 +49,30 @@ <h2 class="pillars-title">
   </div>
 {{ end }}
 
+{{ define "playground-section" }}
+<div class="body__contained body__section-nav playground-section" tabindex="-1">
+  <h1>Try Beam Playground</h1>
+  <p>Beam Playground is an interactive environment to try out Beam transforms and examples without having to install Apache Beam in your environment.
+    You can try the Apache Beam examples at <a href="https://play.beam.apache.org/">Beam Playground (Beta)</a>.
+  </p>
+  <br>
+  <br>
+  <a class="playground__mobile" href="https://play.beam.apache.org/">
+    <img src="images/playground.png" alt="beam playground">
+  </a>
+  <div id="playgroundIframeContainer">
```

Comment:
Why not use a shortcode here?

Reply:
as far as I understood according to the answer, we are not able to use shortcodes: https://discourse.gohugo.io/t/shortcode-inside-define-block/39410